src/extrovermectin.ts

```diff
@@ -81,8 +81,10 @@ export function doingExtrovermectin(): boolean {
 export function crateStrategy(): "Sniff" | "Saber" | "Orb" | null {
   if (!doingExtrovermectin()) return null;
   if (
-    have($skill`Transcendent Olfaction`) &&
-    (property.getString("olfactedMonster") === "crate" || get("_olfactionsUsed") < 2)
+    (have($skill`Transcendent Olfaction`) &&
+      (property.getString("olfactedMonster") === "crate" || get("_olfactionsUsed") < 2)) ||
+    (have($skill`Long Con`) &&
```

Comment:
aw yeah we're conning the fuck outta these crates

Reply:
Having checked in the log, it only cast it once, on garbage tourists. did 0 extrovermectin, so cast it not at all on the crates.